pkg/engine/diff.go

```diff
@@ -291,7 +291,7 @@ func GetResourceOutputsPropertiesString(
 		// Print this property if it is printable and either ins doesn't have it or it's different.
 		if outputDiff != nil || shouldPrintPropertyValue(out, true) {
 			print := true
-			if in, has := ins[k]; has {
```

Comment:
can this have a comment why we don't want this during a refresh.  It's not immediately obvious to m.e

Reply:
Done